test/convnets.jl

```diff
@@ -6,120 +6,114 @@
     _gc()
 end
 
-@testset "VGG" begin
-    @testset "VGG($sz, batchnorm=$bn)" for sz in [11, 13, 16, 19], bn in [true, false]
-        m = VGG(sz, batchnorm = bn)
-        @test size(m(x_224)) == (1000, 1)
-        if (VGG, sz, bn) in PRETRAINED_MODELS
-            @test acctest(VGG(sz, batchnorm = bn, pretrain = true))
-        else
-            @test_throws ArgumentError VGG(sz, batchnorm = bn, pretrain = true)
-        end
-        @test gradtest(m, x_224)
-        _gc()
+@testset "VGG" begin @testset "VGG($sz, batchnorm=$bn)" for sz in [11, 13, 16, 19],
+                                                            bn in [true, false]
+
+    m = VGG(sz; batchnorm = bn)
+    @test size(m(x_224)) == (1000, 1)
+    if (VGG, sz, bn) in PRETRAINED_MODELS
+        @test acctest(VGG(sz; batchnorm = bn, pretrain = true))
+    else
+        @test_throws ArgumentError VGG(sz, batchnorm = bn, pretrain = true)
     end
-end
+    @test gradtest(m, x_224)
+    _gc()
+end end
```

Comment:
Has anyone figured out how to stop the formatter from doing this?

Reply:
Nope, the issue is still open - https://github.com/domluna/JuliaFormatter.jl/issues/604
and as of now, I don't see any [option](https://github.com/domluna/JuliaFormatter.jl/blob/master/src/options.jl) in JuliaFormatter to turn it off.

Reply:
What I did was simply run the formatter only in the `src` directory, and exclude everything else. It's a hack for now but it does most of the job